parachain-template/runtime/src/lib.rs

```diff
@@ -763,6 +763,7 @@ impl_runtime_apis! {
 
 			list_benchmark!(list, extra, frame_system, SystemBench::<Runtime>);
 			list_benchmark!(list, extra, pallet_balances, Balances);
+			list_benchmark!(list, extra, pallet_session, SessionBench::<Runtime>);
```

Comment:
I presume this should be in here @bkchr ?

Reply:
Probably. The template wasn't created by me.